django_advanced_queries/covid_19/queries.py

```diff
@@ -0,0 +1,39 @@
+from django.db import models
+from django.db.models import Count, Max, Avg, F, Subquery, OuterRef, Q
+
+
+class PatientManager(models.Manager):
+    def filter_by_examinations_results_options(self, results):
+        return self.filter(medical_examination_results__result__in=results)
```

Comment:
would be better to add `*args` and `**kwargs` so the support extra filters if needed for example
```
Patient.object.filter_by_examinations_results_options(results=results, person__age__gte=30)
```
(I'm 99% sure that you can't add `filter` after `distinct` so that's the reason why adding args and kwargs will be useful)

Reply:
Done